Composer/packages/lib/shared/src/lgUtils/parsers/extractLgTemplateNames.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+import { LgTemplateName } from '../models/stringTypes';
+
+import { LgTemplateRefPattern } from './lgPatterns';
+
+/**
+ *
+ * @param text string
+ * -[Greeting], I'm a fancy bot, [Bye] ---> ['Greeting', 'Bye']
+ *
+ */
+export default function extractLgTemplateNames(text: string): LgTemplateName[] {
```

Comment:
I think we don't need this extractLgTemplateNames.

We can do extractLgTemplateRefs?